_toc.yml

```diff
@@ -0,0 +1,16 @@
+format: jb-book
+root: README
+parts:
+  - caption: Demo 1 - ETL & Dashboarding 
```

Comment:
I think it would be better to add the content from the readme here as well because those links are taking us back to the Github repository. i.e. 

```

1. Ingest raw data from S3 as tables on Trino

2. Run SQL queries from a Jupyter Notebook environment

3. Demo 1 Elyra Pipeline

4. Results visualized on a Superset Dashboard

5. Video on creating Elyra Pipelines and Superset Dashboard

```

Reply:
will do! 👍